docs/README.md

```diff
@@ -71,47 +71,47 @@ Legend:
-| Digital Workspace   | 4.0.0          | 3.1.0          | 2.9.0          | 2.6.0          | 2.1.0          | N/A           |
-| Transform Router    | 2.1.0          | 2.0.0          | 1.5.3          | 1.5.1          | 1.3.2          | N/A           |
-| Shared File Store   | 2.1.0          | 2.0.0          | 0.16.1         | 0.16.1         | 0.13.0         | N/A           |
-| Sync Service        | 3.9.0          | 3.8.0          | 3.6.0          | 3.5.0          | 3.4.0          | N/A           |
+The table below shows the version of the components deployed by the playbook for ACS 7.x onward.
+
+| Component           | 23.1 Enterprise (Community) | 7.4 Enterprise | 7.3 Enterprise | 7.2 Enterprise | 7.1 Enterprise | 7.0 Enterprise |
+|---------------------|-----------------------------|----------------|----------------|----------------|----------------|----------------|
+| OpenJDK             | 17.0.9                      | 17.0.3         | 17.0.3         | 11.0.15        | 11.0.15        | 11.0.15        |
+| Apache Tomcat       | 10.1.13                     | 9.0.59         | 9.0.59         | 9.0.59         | 9.0.59         | 8.5.76         |
+| PostgreSQL          | 14.x                        | 14.x           | 14.x           | 13.x           | 13.x           | 13.x           |
+| Apache ActiveMQ     | 5.18.3                      | 5.16.6         | 5.16.6         | 5.16.6         | 5.16.6         | 5.16.6         |
+| Repository          | 23.1.0                      | 7.4.0          | 7.3.0          | 7.2.1          | 7.1.1          | 7.0.1.4        |
+| Share               | 23.1.0                      | 7.4.0          | 7.3.0          | 7.2.1          | 7.1.1          | 7.0.1.4        |
+| Search Services     | 2.0.8.2                     | 2.0.6.1        | 2.0.5          | 2.0.4          | 2.0.2          | 2.0.1.1        |
+| Search Enterprise   | 4.0.0 (n/a)                 | 3.3.0          | 3.2.0          | 3.1.1          | 3.1.1          | -              |
+| All-In-One T-Engine | 5.0.0                       | 3.1.0          | 3.0.0          | 2.5.7          | 2.5.6          | 2.3.10         |
+| AOS                 | 2.0.0                       | 1.6.0          | 1.5.0          | 1.4.1          | 1.4.0          | 1.4.0          |
+| Google Docs         | 4.0.0                       | 3.4.0          | 3.3.0          | 3.2.2          | 3.2.1          | 3.2.1          |
+| Digital Workspace   | 4.3.0 (n/a)                 | 4.0.0          | 3.1.0          | 2.9.0          | 2.6.0          | 2.1.0          |
+| Transform Router    | 4.0.0 (n/a)                 | 2.1.0          | 2.0.0          | 1.5.3          | 1.5.1          | 1.3.2          |
+| Shared File Store   | 4.0.0 (n/a)                 | 2.1.0          | 2.0.0          | 0.16.1         | 0.16.1         | 0.13.0         |
+| Sync Service        | 4.0.0 (n/a)                 | 3.9.0          | 3.8.0          | 3.6.0          | 3.5.0          | 3.4.0          |
 
```

Comment:
@anxumalo I've seen a first doc [PR](https://github.com/Alfresco/docs-alfresco/pull/1198) coming through but I think this can help you finalize the update of the stack for Ansible